### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by georgengelmann** [acceptbitcoincash/acceptbitcoincash#1703]
The operator of acceptbitcoin.cash adds affiliate links instead of normal links for some services, if there is some income from referrals, the income is used to pay for hosting and to reward contributors (new additions, fixes, removals) some BCH fractions

**Comment by kenman345** [comment]
We have discussed this in the past. Our hosting is donated by a generous member of the community and our domain covered for the time being by donations we already received. We have enough operating expenses covered for the next 2 years I believe and so we want to avoid any of that sort of behavior regarding referrals if we can. We understand it’s legitimate way and we could disclose that in the section but it’s simply not something we are needing to do, and this we want to avoid making the directory anything but objective and useful for everyone.

That said, it would be very nice to have some help with the site. I created our bot to help reduce the effort in maintaining the site to as low as possible. The amount of effort taken to add a listing is drastically reduced now and I plan to start seeking help shortly. If we need to offer rewards we can evaluate that when we see the response we get back from potential contributors to this project

**Comment by georgengelmann** [comment]
Donated hosting is great, but updated content is very important (we don't want broken links or outdated services listed and new merchants could be added faster) - some incentive to make good contributions could be very helpful - people are completing tasks on LazyFox for a fractions of a BCH

**Comment by kenman345** [comment]
Wanna hit me up on the discord and chat sometime about how to work on some sort of payouts? Would need to know more before we can even consider it
